tests/wpt/tests/webstorage/symbol-props.window.js

```diff
@@ -39,10 +39,10 @@
         Object.defineProperty(storage, key, { "value": "test", "configurable": false });
         assert_equals(storage[key], "test");
         var desc = Object.getOwnPropertyDescriptor(storage, key);
-        assert_true(desc.configurable, "configurable");
+        assert_equals(desc.configurable, false);
```

Comment:
WPT tests are common across all browsers. For example, if you want to see how Chrome is performing on a specific test, you can visit [WPT](https://wpt.fyi/results/webstorage/symbol-props.window.html?label=master&label=experimental&aligned&q=symbol-props.window). Therefore, we need to fix our implementation.

Reply:
Good to know. Thanks!